build/Helix/OutputTestResults.ps1

```diff
@@ -5,7 +5,17 @@ $azureDevOpsRestApiHeaders = @{
 
 Write-Host "Checking test results..."
 
-$testRuns = Invoke-RestMethod -Uri "https://dev.azure.com/ms/microsoft-ui-xaml/_apis/test/runs?buildUri=$($env:BUILD_BUILDURI)" -Method Get -Headers $azureDevOpsRestApiHeaders
+$collectionUri = $env:SYSTEM_COLLECTIONURI 
+$teamProject = $env:SYSTEM_TEAMPROJECT
+
+Write-Host "collectionUri = $collectionUri"
+Write-Host "teamProject = $teamProject"
+
+$baseUri = $collectionUri + $teamProject
+$queryUri = "$baseUri/_apis/test/runs?buildUri=$($env:BUILD_BUILDURI)"
+Write-Host "queryUri = $queryUri"
+
+$testRuns = Invoke-RestMethod -Uri $queryUri -Method Get -Headers $azureDevOpsRestApiHeaders
```

Comment:
These are both building the same URI, as far as I can tell - could we abstract this to a function that both could reference?

Reply:
I've added a new AzurePipelinesHelperScripts.ps1 to let common logic be shared across scripts.

Reply:
Thanks!